asciidoctorj-arquillian-extension/build.gradle

```diff
@@ -0,0 +1,28 @@
+dependencies {
+  compile project(path: ':asciidoctorj')
+  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
+  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
+}
+
+jar.dependsOn project(':asciidoctorj').testClasses
+
+jar {
+  from project(':asciidoctorj').sourceSets.test.output
+  include 'org/asciidoctor/arquillian/**/*'
```

Comment:
> I am a little disappointed that you always need some extra plugins to fully cover Maven's possibilities.

My feeling exactly. I love Gradle's power and I think it is fully capable of making tasks like this easy...but it just seems to be left as unfinished work. The best thing we can do for Gradle (and ourselves) is to at least make the core engineers aware of these shortcomings (in a blog, forum post, etc). If they continue to ignore the gravity of the situation, then we can let the disappointment show through.